underfs/wasb/src/main/java/alluxio/underfs/wasb/WasbUnderFileSystem.java

```diff
@@ -67,7 +77,7 @@ public static Configuration createConfiguration(UnderFileSystemConfiguration con
    */
   public static WasbUnderFileSystem createInstance(AlluxioURI uri,
       UnderFileSystemConfiguration conf) {
-    Configuration wasbConf = createConfiguration(conf);
+    Configuration wasbConf = createConfiguration(conf, uri.getScheme().startsWith("wasb://"));
```

Comment:
Shouldn't this be `startsWith("wasbs://")`?

Instead, can we have 2 other constants, like `SCHEME_SECURE="wasbs://"`, and `SCHEME_INSECURE="wasb://"`?

Reply:
Changed and verified